bridges/TwitterBridge.php

```diff
@@ -232,9 +232,42 @@ public function collectData(){
 			// extract tweet timestamp
 			$item['timestamp'] = $tweet->created_at;
 
-			// generate the title
+			// Convert plain text URLs into HTML hyperlinks
 			$item['title'] = $tweet->full_text;
-			$cleanedTweet  = $tweet->full_text;
+			$cleanedTweet = $tweet->full_text;
+			$foundUrls = false;
+
+			if (isset($tweet->entities->media)) {
+				foreach($tweet->entities->media as $media) {
+					$cleanedTweet = str_replace($media->url,
+						'<a href="' . $media->expanded_url . '">' . $media->display_url . '</a>',
+						$cleanedTweet);
+					$item['title'] = str_replace($media->url,
+						$media->expanded_url,
+						$item['title']);
```

Comment:
Following would be enough after $cleanedTweet is ready
```php
$item['title'] = strip_tags($cleanedTweet);
```

Reply:
ok